lib/utility/convertJSDateToJSForceDate.js

```diff
@@ -0,0 +1,19 @@
+var jsforce = require('jsforce');
+
+/**
+ * Converts JS Date Objects to JS Force Date Literals.
+ * @param {*}
+ * @returns object
+ */
+exports.convertJSDateToJSForceDate = function convertJSDateToJSForceDate(obj) {
+	if (obj instanceof Date) {
+		return jsforce.Date.toDateLiteral(obj);
+	} else if (typeof(obj) === 'string') {
+		return jsforce.Date.toDateLiteral(Date.parse(obj));
+	} else if (typeof(obj) === 'object') {
```

Comment:
Hi Jan,
Do you mean this condition here 

`else if (typeof(obj) === 'string') {`

If so, it's just for precaution. I think it won't be necessary.

About `else if (typeof(obj) === 'object') {` it's doing recursive data converting.